samples/project-anfield/import_cluster_config.py

```diff
@@ -625,7 +716,7 @@ def create_protection_sources():
 
         for source in sources:
             environment = source.protection_source.environment
-            if environment not in env_list:
+            if environment not in [env_enum.K_VMWARE]: #env_list:
```

Comment:
Make sure this doesn't regress other env type, why did we restrict it to vmware only ?